core/templates/components/state-editor/state-content-editor/state-content-editor.directive.html

```diff
@@ -50,6 +50,7 @@
   </div>
```

Comment:
We should probably still tell the user why we are disabling the Save button. Show a message that says to shorten the card or break it up into 2 cards.

Reply:
Sure, I'll update you on the same.
Thanks, @EricZLou.

Reply:
Hello, Sorry for the late reply @EricZLou.
Here is the SS
![Screenshot from 2021-10-02 21-44-37](https://user-images.githubusercontent.com/44300735/135726904-1de037ac-5bc2-44a3-9ae7-df9d0b8d34dc.png)